src/cmd/isolated_server.cpp

```diff
@@ -74,7 +74,7 @@ void help(const char* argv[]) {
 int main(int argc, const char* argv[]) {
   string accountJsonFilePath;
   uint port{5555};
-  string blocknum_str{"0"};
+  string blocknum_str{"1"};
```

Comment:
Be aware that reverting this will break the js test getBlockByNumber. So far I understand blocks, also in production, start at '0'.